cli/cmd/root.go

```diff
@@ -48,6 +48,7 @@ func NewRootCmd() *cobra.Command {
 
 	rootCmd.PersistentFlags().Bool("debug", false, "enable debug logging")
 	rootCmd.PersistentFlags().Bool("force", false, "disable version compatibility checks - might result in corrupted clusters")
+	rootCmd.PersistentFlags().String("tf-log", "NONE", "sets the Terraform log level (default \"NONE\" - no logs)")
```

Comment:
Since only (iam) create and terminate, and mini-Constellation actually use terraform, do we actually want this as a global flag?

Reply:
We can do that, but as 4 commands were using it i was unsure where to put it